src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
Sorry for the hurried response; I originally had a larger but ill-focused response.  So here are the specific quotes where new (and IMHO relevant) conversation topics were introduced:

> I am just a bit confused about the state of this PR. Is it accepted, rejected, or deferred until concerns about functionality/performance are addressed? I see neither of these, but just some ideas for further improvements (which are interesting, but the evaluation of the PR should not depend on those results unless they reveal a bug or the like).

> Fearing not to be able to improve further should not be a reason to keep you from improving, and by the way, why it is important that you improve over it with a similar patch?

FYI (looking forward), I don't think I'd reject a PR without first discussing reasons (such as criteria) to consider rejecting it (hopefully historically I've been consistent in this; if not I should become consistent with it in the future).